src/gestalts/GestaltGraph.ts

```diff
@@ -28,6 +29,11 @@ import * as gestaltsErrors from './errors';
 import * as gestaltsEvents from './events';
 import * as aclUtils from '../acl/utils';
 import { never } from '../utils';
+import * as utils from '../utils';
+
+// TODO: add tracking for the oldest claimID for a node.
+//  This needs to be removed if the node is removed.
+//  ALso possible undefined.
```

Comment:
Todo?? No todos remember! They need to be issues.

Reply:
My bad, I missed that in clean up. I'll push a fix to staging in a moment.